core/src/main/java/org/apache/james/core/User.java

```diff
@@ -23,12 +23,17 @@
 import java.util.Objects;
 import java.util.Optional;
 
+import org.apache.james.mailbox.model.MailboxConstants;
+
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
 import com.google.common.base.Strings;
 
 public class User {
+
+    private static final String DEFAULT_DELIMITER = String.valueOf(MailboxConstants.DEFAULT_DELIMITER);
```

Comment:
we should make sure Default is the only possible value in James and forbid others, don't you think ?

Reply:
I'm on it, put it's triggering a lot of modifications.
I will propose another commit for that.